openmdao/docs/usr-guide/mpi_windows.rst

```diff
@@ -0,0 +1,232 @@
+.. _MPI on Windows:
+
+===============
+MPI on Windows
+===============
+
+This document provides the setup and usage of MPI (Message Passing Interface) in OpenMDAO on Windows (64-bit) using Python3 (64-bit). 
+We start with installing the necessary packages and test them to make sure they work.
+Then we will look at a simple example of how to take advantage of MPI in OpenMDAO.
+
+Python 3
+------------
+
+The Python 3 Environment should be installed from either the `official Python page`_ or Anaconda_.  Python 3.5 and later use Microsoft Visual Studio 2015.  Python 3.3 and 3.4 use Microsoft Visual Studio 2010.  The version you install must match the Visual Studio environment you install.  Make sure you also install `numpy` and `scipy` with your environment.
+
+.. _official Python page: https://www.python.org/downloads/windows/
+.. _Anaconda: https://www.continuum.io/downloads
+
+
+Microsoft Visual Studio
+-------------------------
+
+Visual Studio 2010 Professional was used during this tutorial as the C/C++ compiler, because the official Python 3.4 build was compiled with that version.  Python2 cannot be used
+because it was built with Visual Studio 2008 which no longer exists.
```

Comment:
Have you seen this?  I haven't tried it yet, it does look like a version of Visual Studio is available for python 2.7.  https://www.microsoft.com/en-us/download/details.aspx?id=44266